CITATION.cff

```diff
@@ -1,20 +1,41 @@
 cff-version: 1.2.0
-message: "If you use this software, please cite it as below."
+message: |
+  Please cite this software using these metadata. "*" indicates co-equal authorship.
+
+title: "QGreenland"
+doi: "10.5281/zenodo.4558266"
+abstract: "This software generates the QGreenland data package for QGIS."
+contact:
+  - name: "The QGreenland Team"
+    email: "qgreenland.info@gmail.com"
+  - name: "The QGreenland community" 
+    website: "https://github.com/nsidc/qgreenland/discussions"
+type: "software"
+license: ["MIT"]
+keywords:
+  - "GIS"
+  - "QGIS"
+  - "Python"
+  - "education"
+
+url: "https://github.com/nsidc/qgreenland"
+repository-code: "https://github.com/nsidc/qgreenland"
+
+version: "3.0.0alpha4"
+date-released: "2023-08-23"
+
 authors:
-- family-names: "Moon"
-  given-names: "Twila"
-  orcid: "https://orcid.org/0000-0003-0968-7008"
-- family-names: "Fisher"
+- family-names: "*Fisher"
```

Comment:
Putting the asterisk in the `family-names` attribute seems to work well in the citations generated by GitHub. Hopefully Zenodo will also be OK with it and the work will be correctly link to our orcids.